src/features/events_list/atoms/currentEventBbox.ts

```diff
@@ -33,7 +33,7 @@ export const currentEventBbox = createAtom(
     const bbox = eventData.bbox;
 
     onAction('fitBounds', () => {
-      currentMapPositionAtom.setCurrentMapBbox.dispatch(bbox);
+      setCurrentMapBbox(store.v3ctx, bbox);
```

Comment:
see https://github.com/konturio/reatom-analyzer/blob/main/docs/reatom2-atom.md#basic-atom-structure
v3ctx is available in reducer, store not needed

Reply:
Fixed